### PR TITLE
Update redis package

### PR DESCRIPTION
**Pull request by benrudolph**
(opening for tests)

### CODE REVIEW

File: custom/bihar/utils.py [hunk 1]
```diff
@@ -1,7 +1,7 @@
 from operator import attrgetter
 from django.utils.translation import ugettext_noop
 from corehq.apps.groups.models import Group
-from redis_cache.cache import RedisCache
+from django_redis.cache import RedisCache
```
Comment (dimagimon) [inline custom/bihar/utils.py]: F401 'RedisCache' imported but unused

**Comment by benrudolph** [comment]
woot :white_check_mark:!
corresponding PRs:
https://github.com/dimagi/pillowtop/pull/121
https://github.com/dimagi/dimagi-utils/pull/301
fyi @snopoke i think i'll put this on staging next week
buddy @czue

**Comment by snopoke** [comment]
👍

On 4 August 2015 at 16:48, Ben <notifications@github.com> wrote:

> woot [image: :white_check_mark:]!
> corresponding PRs:
> dimagi/pillowtop#121 <https://github.com/dimagi/pillowtop/pull/121>
> dimagi/dimagi-utils#301 <https://github.com/dimagi/dimagi-utils/pull/301>
> fyi @snopoke <https://github.com/snopoke> i think i'll put this on
> staging next week
> buddy @czue <https://github.com/czue>
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/dimagi/commcare-hq/pull/7745#issuecomment-127638938>.
>



-- 
Simon Kelly
Senior Engineer | Dimagi South Africa

File: settingshelper.py [hunk 2]
```diff
@@ -112,6 +112,6 @@ def get_extra_couchdbs(config, couch_database_url):
 
 def celery_failure_handler(task, exc, task_id, args, kwargs, einfo):
     from redis.exceptions import ConnectionError
-    from redis_cache.exceptions import ConnectionInterrumped
-    if isinstance(exc, (ConnectionInterrumped, ConnectionError)):
+    from django_redis.exceptions import ConnectionInterrupted
```
Comment (czue) [inline settingshelper.py]: :disappointed:

**Comment by czue** [comment]
sweet!

File: settingshelper.py (hunk 2 above)
Comment (benrudolph) [inline settingshelper.py]: hahah

**Comment by benrudolph** [comment]
This passed QA

File: hqscripts/generic_queue.py [hunk 3]
```diff
@@ -63,11 +63,7 @@ def get_redis_client(self):
         rcache = cache_core.get_redis_default_cache()
         if not isinstance(rcache, RedisCache):
             raise RedisClientError("Could not get redis connection.")
-        try:
-            client = rcache.raw_client
-        except:
-            raise RedisClientError("Could not get redis connection.")
-        return client
+        return rcache
```
Comment (gcapalbo) [inline hqscripts/generic_queue.py]: how come this was removed - does rcache now represent the raw client?

**Pull request merged**